controllers/iamrole_controller.go

```diff
@@ -298,6 +298,21 @@ func (r *IamroleReconciler) ConstructCreateIAMRoleInput(ctx context.Context, iam
 		tags["Cluster"] = config.Props.ClusterName()
 	}
 
+	// Custom tags value should be a string of comma seperated key/value pairs
+	// example annotation: "iamroles.iammanager.keikoproj.io/tags": "key1=value1,key2=value2"
+	if customTagsString, ok := iamRole.GetAnnotations()["iamroles.iammanager.keikoproj.io/tags"]; ok {
```

Comment:
There is a common utility function  to read the annotation. Please use that

Reply:
Sure, will change this